### PR TITLE
pathogen field support for PhyloRun, Sample, PathogenGenome tables

### PR DESCRIPTION
**Pull request by phoenixAja**
### Summary:
- **What:** add pathogen foreign keys to Sample, PthogenGenome, and PhyloRun tables
- **Ticket:** [sc211473](https://app.shortcut.com/genepi/story/211473)
- **Env:** `<rdev link>`

### Checklist:
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/database_migrations/versions/20220830_171547_add_pathogen_field.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+revision = "20220830_171547"
+down_revision = "20220816_000535"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    op.add_column(
+        "pathogen_genomes",
```
Comment (jgadling) [inline src/backend/database_migrations/versions/20220830_171547_add_pathogen_field.py:20]: I think our data model had pathogen_id added to samples, I don't think we need it added to pathogen_genomes as well.

File: src/backend/database_migrations/versions/20220830_171547_add_pathogen_field.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        "pathogens",
+        ["pathogen_id"],
+        ["id"],
+        source_schema="aspen",
+        referent_schema="aspen",
+    )
+
+    op.add_column(
+        "phylo_runs",
```
Comment (jgadling) [inline src/backend/database_migrations/versions/20220830_171547_add_pathogen_field.py:35]: We also need to add a pathogen_id to phylo_trees

**Review by jgadling — CHANGES_REQUESTED** [review]
We should merge & deploy migrations before updating our SA models to use the new columns, or else we'll cause errors while upgrading our app.

**Comment by vincent-czi** [comment]
> We should merge & deploy migrations before updating our SA models to use the new columns, or else we'll cause errors while upgrading our app.

I agree with this, but how do we handle the Prod aspect? Would we make a separate PR of just the migrations, merge that to Staging, verify things didn't break in Staging and the migration succeeded, then do a Prod push before coming back with the PR to handle the SA models and rest of the changes? Or are you thinking something different?

**Comment by jgadling** [comment]
> I agree with this, but how do we handle the Prod aspect? Would we make a separate PR of just the migrations, merge that to Staging, verify things didn't break in Staging and the migration succeeded, then do a Prod push before coming back with the PR to handle the SA models and rest of the changes? Or are you thinking something different?

Yep, we can deploy to prod once we've verified that staging looks good, and then open up a PR for other code changes that use new fields.

**Comment by phoenixAja** [comment]
ok i'll close this and reopen the migration/ sqlalchemy support separately

**Pull request closed without merging**